activities/member-relations/collect-information.md

```diff
@@ -0,0 +1,32 @@
+---
+type: Resource
+---
+
+# What information to collect about members for member relations 
+
+## Introduction
+
+This resource outlines the information that Foundation for Public Code staff use to map member relations.
+
+The information collected will be used to structure and keep track of our ongoing relationships with members. It is based on our [mission](../../organization/mission.md) and the first iterations of the [founding member proposal template](founding-membership-proposal.md). In line with our communication principles, this should be concise and hypertextual where possible. 
```

Comment:
To avoid content debt (future rework you already know about), put this in the present tense now. It will be true as soon as we start doing any of this.

```suggestion
The information collected is used to structure and keep track of our ongoing relationships with members. It is based on our [mission](../../organization/mission.md) and the first iterations of the [founding member proposal template](founding-membership-proposal.md). In line with our communication principles, it is concise and hypertextual where possible. 
```